aws/resource_aws_instance_test.go

```diff
@@ -1288,6 +1288,108 @@ func TestAccAWSInstance_associatePublic_overridePrivate(t *testing.T) {
 	})
 }
 
+func TestAccAWSInstance_getPasswordData_true(t *testing.T) {
```

Comment:
Can we reduce this down to two acceptance tests please? false -> true and true -> false should be sufficient